### ISSUE TITLE
trasa-reflex, continued maintainership?

### ISSUE THREAD

**Issue by chessai** [haskell-trasa/trasa#14]
i know that, given we no longer use reflex at work, @andrewthad, @goolord, and myself will have a harder time keeping up with `trasa-reflex`. i don't know if @mckeankylej or any of us are willing to continue maintaining it. what are your thoughts?

**Comment by chessai** [comment]
to be clear, what i'm saying is that if no one is willing to maintain it, it should probably be deprecated.

**Comment by goolord** [comment]
i think it's fine to just leave it, if someone wanted to use it it would probably be easily updated

**Comment by chessai** [comment]
alright, but perhaps it should be documented in the README that it is not maintained

**Comment by aljce** [comment]
I don’t think it matters very much but I would lean on the side of depreciation. The problem isn’t with the code but maintaining the nix. Reflex platform changes often and it can be hard to figure out what needs to be updated to get everything to build.
